posthog/hogql_queries/web_analytics/test/test_web_analytics_query_runner.py

```diff
@@ -99,3 +101,14 @@ def test_sample_rate_cache_key_changes_with_date_range(self):
         key_b = self._create_web_stats_table_query(date_from_b, date_to, properties)._sample_rate_cache_key()
 
         self.assertNotEquals(key_a, key_b)
+
+    def test_sample_rate_from_count(self):
+        self.assertEqual(SamplingRate(numerator=1), _sample_rate_from_count(0))
+        self.assertEqual(SamplingRate(numerator=1), _sample_rate_from_count(1_000))
+        self.assertEqual(SamplingRate(numerator=1), _sample_rate_from_count(10_000))
+        self.assertEqual(SamplingRate(numerator=1, denominator=10), _sample_rate_from_count(100_000))
+        self.assertEqual(SamplingRate(numerator=1, denominator=10), _sample_rate_from_count(999_999))
+        self.assertEqual(SamplingRate(numerator=1, denominator=100), _sample_rate_from_count(1_000_000))
+        self.assertEqual(SamplingRate(numerator=1, denominator=100), _sample_rate_from_count(9_999_999))
+        self.assertEqual(SamplingRate(numerator=1, denominator=1000), _sample_rate_from_count(10_000_000))
+        self.assertEqual(SamplingRate(numerator=1, denominator=1000), _sample_rate_from_count(99_999_999))
```

Comment:
Not a biggie by any means, but I was told to use the `assert` keyword instead of these methods from the base class, e.g:

```python
assert SamplingRate(numerator=1, denominator=1000) == _sample_rate_from_count(99_999_999)
```

Reply:
ah fair, I had not heard this